### PR TITLE
installing smtp

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
         "drupal/core": "8.7.0",
         "drupal/devel": "^2.1",
         "drupal/easy_breadcrumb": "^1.12",
+        "drupal/helper": "^1.3",
```
Comment (ElXyfer) [inline composer.json:33]: You don't need the helper module anymore remove this please.

File: config/sync/core.extension.yml [hunk 2]
```diff
@@ -32,6 +32,7 @@ module:
   file: 0
   filter: 0
   help: 0
+  helper: 0
```
Comment (ElXyfer) [inline config/sync/core.extension.yml:35]: Don't forget to remove this too.

File: composer.json (hunk 1 above)
Comment (katlazarova) [reply composer.json]: done.

File: config/sync/core.extension.yml (hunk 2 above)
Comment (katlazarova) [reply config/sync/core.extension.yml]: done.

**Pull request merged**